### PR TITLE
Ctr plot

### PR DESCRIPTION
**Pull request by taweihuang**
1. Fix typo - "tunning" -> "tuning"

2. Refine the name:
"seq_error" -> "cum_regret"
"data_simulation" -> "simulate_data"
"policy_evaluation" -> "evaluate_policy"
"tunning_plot" -> "plot_tuning_curve"

3. Add rewardplot.py => new methods for plot regret bound analysis w.r.t. time

4. Delete unnecessary functions "regret_calculation" and "regret_plot" om simulation.py

### CODE REVIEW

File: striatum/rewardplot.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            The bandit algorithm you want to evaluate.
+    """
+
+    avg_reward = calculate_avg_reward(policy)
+    plt.plot(avg_reward.keys(), avg_reward.values(), 'r-', label="average reward")
+    plt.xlabel('time')
+    plt.ylabel('avg reward')
+    plt.legend()
+    plt.title("Average Reward with respect to Time")
+    plt.show()
```
Comment (ianlini) [inline striatum/rewardplot.py]: `show` should be called manually by the user.
Directly using plt is not good here, but we can fix this in the next PR.
You can refer to pandas such as adding the ax as a parameter and use that ax to plot.

File: striatum/rewardplot.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            The bandit algorithm you want to evaluate.
+    """
+
+    avg_reward = calculate_avg_reward(policy)
+    plt.plot(avg_reward.keys(), [1.0 - reward for reward in avg_reward.values()], 'r-', label="average regret")
+    plt.xlabel('time')
+    plt.ylabel('avg regret')
+    plt.legend()
+    plt.title("Average Regret with respect to Time")
+    plt.show()
```
Comment (ianlini) [inline striatum/rewardplot.py]: same as above

Comment (taweihuang) [inline striatum/rewardplot.py]: How about return a figure? Like this

`    avg_reward = calculate_avg_reward(policy)
    fig = plt.plot(avg_reward.keys(), [1.0 - reward for reward in avg_reward.values()],
                   'r-', label="average regret")
    fig = plt.xlabel('time')
    fig = plt.ylabel('avg regret')
    fig = plt.legend()
    fig = plt.title("Average Regret with respect to Time")
    return fig
`

Comment (ianlini) [inline striatum/rewardplot.py]: returning figure is not very meaningful because the user can get it from `plt`

Comment (ianlini) [inline striatum/rewardplot.py]: remove the `plt.show()` first so I can merge this PR

**Pull request merged**